packages/node_modules/@node-red/editor-client/src/js/history.js

```diff
@@ -15,59 +15,89 @@
  **/
 RED.history = (function() {
     var undo_history = [];
+    var redo_history = [];
 
     function undoEvent(ev) {
         var i;
         var len;
         var node;
         var subflow;
         var modifiedTabs = {};
+        var inv_ev;
```

Comment:
For future reference, we do not use [Snake Case](https://en.wikipedia.org/wiki/Snake_case) with variables in this project. Some may exist, but I want to avoid adding more (like `undo_history`... oops, missed that one)

So instead of `inv_ev` it ought to be either `invEv` or `inverseEv` to be clearer.

Reply:
OK, I'll change `inv_ev` and `redo_history` to `inverseEv` and `redoHistory` respectively.
`undo_history` remains intact.  May I change it to `undoHistory`?

Reply:
I am not requesting you change this today - unless you want to do it. But I wanted to highlight this coding style for future work. If you do want to update this PR, then `undo_history` can also be changed.

Reply:
For consistency, I updated to `inverseEv`, `redoHistory`, and `undoHistory` in a recent commit.